### ISSUE TITLE
Cant get the Content of the HttpResponse in the Interceptor

### ISSUE THREAD

**Issue by TheSwerik** [jsakamoto/Toolbelt.Blazor.HttpClientInterceptor#3]
I cant get the Content of the Response in the Interceptor.

Code Examples:

```csharp
protected override void OnInitialized() { HttpClientInterceptor.AfterSend += Interceptor_AfterSend; }

private async void Interceptor_AfterSend(object sender, HttpClientInterceptorEventArgs e)
{
        Console.WriteLine(await e.Response.Content.ReadAsStringAsync());
}
```

In the Service:
```csharp
var response = await Http.PostAsync(Url, JsonContent.Create(model));
```

Stacktrace:
![stacktrace](https://user-images.githubusercontent.com/47034061/92083949-06686080-edc7-11ea-9216-005048d89b0f.png)

**Comment by jsakamoto** [comment]
@TheSwerik Thank you for your report.

I guess it is not a bug, it may be the limitation of the architecture of the runtime, I think at this time.

Anyway, I'll investigate it and try to come up with any solutions or workarounds.

**Comment by jsakamoto** [comment]
@TheSwerik  Thank you for waiting!

I published version 9.1.0 preview 1 of this package today.

- https://www.nuget.org/packages/Toolbelt.Blazor.HttpClientInterceptor/9.1.0-preview.1

This version would allow you to get the content of the HttpResponse in your "AfterSend" event handler by referencing the return value of calling "GetCapturedContentAsync()". (if I had implemented this feature correctly...)

Code Examples:

```csharp
protected override void OnInitialized() { HttpClientInterceptor.AfterSend += Interceptor_AfterSend; }

private async void Interceptor_AfterSend(object sender, HttpClientInterceptorEventArgs e)
{
        // Don't reference "e.Response.Content" directly to read the content.
        // Instead, please use the return value of the  "GetCapturedContentAsync()" method.
        var capturedContent = await e.GetCapturedContentAsync();
        Console.WriteLine(await capturedContent.ReadAsStringAsync());
}
```

Please see also sample code at the following link.

- https://github.com/jsakamoto/Toolbelt.Blazor.HttpClientInterceptor/blob/master/SampleSite/Client/Program.cs#L48

Could you try it out?

**Comment by TheSwerik** [comment]
Dude this works like a charm
Thank you for fixen that so quickly

I even tried It in the final usecase where I put the content in a Toast and it just works
